src/app/containers/FrontPageSection/index.test.jsx

```diff
@@ -205,10 +205,44 @@ describe('FrontPageSection Container', () => {
       expect(section.getAttribute('aria-labelledby')).toEqual(label.id);
     });
 
+    it('should add styling to visually hide section label when width of screen is less than 600px and sectionNumber === 0', () => {
+      const { container } = render(
+        <ServiceContextProvider service="igbo">
+          <FrontPageSection group={group} sectionNumber={0} />
+        </ServiceContextProvider>,
+      );
+
+      global.innerWidth = 500;
+      global.dispatchEvent(new Event('resize'));
+      const sectionLabel = container.getElementsByTagName('section')[0]
+        .firstChild;
+
+      expect(sectionLabel).toHaveStyleRule('clip-path', 'inset(100%)', {
```

Comment:
Hey, soz I dont think we should have tests for the styles. Instead we should assert that `SectionLabel` was called with `visuallyHidden` true or false